### ISSUE TITLE
Metamask login fails

### ISSUE THREAD

**Issue by rabble** [LibertyDSNP/example-client#98]
Issue description
---------------
When trying to do a new account on this example app with a fresh install it fails to login with metamask. 

"Error registering: DSNPError: Could not find address for BeaconFactory contract."

Reproduce Issue
---------------
1. Fresh install of example-client
2. Go to web interface
3. Login with mask.
4. Attempt to set user name.

Expected result
---------------
- User should be able to create an account and set their username.  

Actual result
---------------
- Error of "Error registering: DSNPError: Could not find address for BeaconFactory contract." is displayed and the user can't use the app. 

Additional details / screenshot
---------------
<img width="522" alt="Screen Shot 2021-09-14 at 11 17 46 AM" src="https://user-images.githubusercontent.com/76/133312004-1b100b8e-87e9-40c4-9613-6c2b3b91c3cb.png">

**Comment by shannonwells** [comment]
Hi there, can you please show what your network config is for connecting?  That usually indicates a network that doesn't have DSNP contracts deployed on it.

**Comment by wilwade** [comment]
@rabble As @shannonwells said. If you were connecting to Mainnet Ethereum, try connecting to Rinkeby instead. We have not deployed the contracts to Mainnet at this time.

**Comment by acruikshank** [comment]
Thanks for sending this issue, @rabble. It's really not good that the default Metamask sign-in in leads to this obscure error on the next screen. We're going to work to make the sign in requirements more obvious before you sign up, and make common error messages more clear.

**Comment by rabble** [comment]
So i'd say two things, one this needs some documentation in the setup reminding folks that the contracts are only deployed to Rinkeby and not maintnet. 

And since it's an example app, put it in the UI as well. 

I got a timeout when trying to use  torus as well. 

Why not run your own hosted version of the example-client with a link? That'd make it easier to play with for folks interested.

**Comment by wilwade** [comment]
Behind on updating this, but a quick start guide was added to the initial page of the example client. #117 and #120 

> Why not run your own hosted version of the example-client with a link? That'd make it easier to play with for folks interested.

We have thought about doing this, but it would be a read-only version (we don't want to have an open upload for example client). We tested deploying to GitHub pages to get this read-only version (#23) Perhaps we should re-evaluate the need to do that.

I'll make a new issue to track that #144
